src/ports.js

```diff
@@ -0,0 +1,15 @@
+function Port (name) {
+    this.ports = name;
+    this.ships = []
+};
+
+Port.prototype = {
+    addShip(ship) {
+    this.ships.push(ship)   
+    },
+    removeShip() {
+       this.ships.pop()
+    },
```

Comment:
Leaving a blank line between methods makes them easier to read at a glance:
```suggestion
    },

    removeShip() {
        this.ships.pop()
    },
```